tests/unit/test_patch.py

```diff
@@ -0,0 +1,204 @@
+import json
+
+from dash import Patch
+from dash._utils import to_json
+
+
+def patch_to_dict(p):
+    return json.loads(to_json(p))
+
+
+def test_pat001_patch_assign_item():
+    p = Patch()
+    p["item"] = "item"
```

Comment:
minor point... but to play devil's advocate these two `"item"` strings should be different.